packages/docs/docs/charting/capturing-vital-signs.md

```diff
@@ -0,0 +1,136 @@
+# Capturing Vital Signs
+
+Vital signs are a group of important medical signs that measure the body's most vital (life-sustaining) functions. The four main vital signs are body temperature, pulse rate, respiration rate, and blood pressure. However, sometimes other measures, such as blood oxygen, weight, and blood glucose level, are included as well.
+
+## The `Obeservation` Resource
+
+Vital signs are stored as [`Observation`](/docs/api/fhir/resources/observation) resources. To track the specific vital that is being observed, use the `Observation.code` element. This describes what was observed, and is sometimes called the observation &quot;name&quot;. This should be coded using [LOINC](https://loinc.org/search) codes per US core guidelines.
+
+To record the actual value of a measurement, you can use one of the various value fields on the `Observation` resource. There are fields for `valueQuantity`, `valueString`, `valueBoolean` and more. To see all of the available fields on the `Observation` resource, see the [`Observation` resource reference](docs/api/fhir/resources/observation).
+
+**Example: **
+
+```ts
+
+{
+  resourceType: 'Observation',
+  id: 'example-observation-1',
+  // ...
+  code: {
+    system: 'http://loinc.org',
+    code: '8310-5',
+    display: 'Body temperature'
+  },
+  valueQuantity: {
+    value: 98.2,
+    unit: 'degrees fahrenheit'
+  }
+}
+
+```
+
+In this example, we use the `code` field to define the vital we are measuring, in this case body temperature. We define it using a LOINC code, as well as a `display`, which is a readable representation of what we are measuring. In addition, since we are measuring temperature, which is easily represented as a number or quantity, we use the `valueQuantity` field to define the measurement result. We can use the `unit` field to provide a readable string of the unit of the measurement.
+
+## Who Performed the `Observation`
+
+Observation data can come from various sources, including the practitioner, a nurse, being self-reported by the patient, or many others. The source of the observation should be recorded on the `Observation.performer` element. This can be important to help provide a degree of confidence in the observation and to inform where any follow-up questions should be directed.
+
+## Medical Devices
```

Comment:
This can be supplanted by the table above.

Keep it simple, and just reference `Device`